### PR TITLE
Rename extension

### PR DESCRIPTION
**Pull request by frostyfan109**

### CODE REVIEW

File: jupyter-config/nb-config/jupyterlab_eduhelx_submission.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "NotebookApp": {
     "nbserver_extensions": {
-      "jupyterlab_eduhelx_submission": true
+      "eduhelx_jupyterlab_student": true
```
Comment (Hoid) [inline jupyter-config/nb-config/jupyterlab_eduhelx_submission.json:4]: You didn't rename this file or the other one in `jupyter-config/server-config/`. I assume that should be fixed?

File: ui-tests/tests/jupyterlab_eduhelx_submission.spec.ts [hunk 2]
```diff
@@ -16,6 +16,6 @@ test('should emit an activation console message', async ({ page }) => {
   await page.goto();
 
   expect(
-    logs.filter(s => s === 'JupyterLab extension jupyterlab_eduhelx_submission is activated!')
+    logs.filter(s => s === 'JupyterLab extension eduhelx_jupyterlab_student is activated!')
```
Comment (Hoid) [inline ui-tests/tests/jupyterlab_eduhelx_submission.spec.ts:19]: You didn't change the name of this file

Comment (frostyfan109) [reply ui-tests/tests/jupyterlab_eduhelx_submission.spec.ts:19]: Mb, I was lazy and didn't realize vscode search doesn't grep filenames. Fixing all the filenames now and another review to make sure I'm not missing stuff.

File: jupyter-config/nb-config/jupyterlab_eduhelx_submission.json (hunk 1 above)
Comment (frostyfan109) [reply jupyter-config/nb-config/jupyterlab_eduhelx_submission.json:4]: Yep

**Comment by frostyfan109** [comment]
Renamed files/directories and did a final scan to make sure all references are caught (hopefully). Still confirming it's working properly.

Edit: working

**Review by Hoid — APPROVED** [review]

**Pull request merged**